edx_exams/apps/api/v1/views.py

```diff
@@ -453,3 +469,19 @@ def put(self, request, attempt_id):
             status=status.HTTP_400_BAD_REQUEST,
             data={'detail': f'Unrecognized action "{action}"'}
         )
+
+    def post(self, request):
+        """
+        HTTP POST handler to create exam attempt
+        """
+        should_start_immediately = request.data.get('start_clock', 'false').lower() == 'true'
```

Comment:
Nice, I like how you handled the case where you want to make an exam attempt started.